flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
this is very sad

Reply:
AFAIK, the `org.freedesktop.Sdk//21.08` does not support i386 for now, thus I cannot build an i386 flatpak by using `--arch=i386`.

Reply:
He means ARM, not 32 bit

Reply:
I mean `aarch64`, so I can run it on a RaspberryPi.